docs/running/cliOptions.rst

```diff
@@ -189,6 +189,11 @@ the logging module:
                         when they are launched in gigabytes. You may want to
                         set this if your jobs require a lot of disk space. The
                         default value is 50.
+  --nodeStorageOverrides NODESTORAGEOVERRIDES
+                        (optional) Comma-separated list of nodeType:nodeStorage that
```

Comment:
We shouldn't say `(optional)` here; it has override in the name and none of the other optional options say `(optional)`.

Reply:
Good point! Removed from here and elsewhere.